MaxText/accelerator_to_spec_map.py

```diff
@@ -29,19 +29,28 @@ class SystemCharacteristics:
   platform: str
   topology_name: Optional[str]
   chip_config_name: Optional[str]  # 'megacore' or 'default'
-  chips_per_host_bounds: Optional[tuple]
+  chips_per_host_bounds: Optional[tuple]  # number of chips on each host in each dimension.
```

Comment:
I think this is actually the chips_per_vm? For example a v6e host has two vms, each with 4 chips in a 2, 2, 1 arrangement.

Reply:
Your comment makes sense, but "chips_per_host_bounds" is something used internally, and my comment comes from there.